setup/upgrade_major.rst

```diff
@@ -124,24 +126,50 @@ done!
 -----------------------------------------------
 
 Once your code is deprecation free, you can update the Symfony library via
-Composer by modifying your ``composer.json`` file:
+Composer by modifying your ``composer.json`` file and changing all the libraries
+starting with ``symfony/`` to the new major version:
 
-.. code-block:: json
+.. code-block:: diff
 
     {
         "...": "...",
 
         "require": {
-            "symfony/symfony": "^4.1",
+-             "symfony/cache": "4.4.*",
++             "symfony/cache": "5.0.*",
+-             "symfony/config": "4.4.*",
++             "symfony/config": "5.0.*",
+-             "symfony/console": "4.4.*",
++             "symfony/console": "5.0.*",
```

Comment:
I thought the diff rormat would be more useful here.

Also, in practical terms, ever since the `extra.symfony.require` key was added, when you intall a Symfony package, it uses the version constraint format `4.4.*` not `^4.4`. And so, for consistency, I'm showing that format and telling people to continue using it.